pkg/skuba/actions/node/upgrade/plan.go

```diff
@@ -33,16 +33,17 @@ func Plan(nodeName string) error {
 	if err != nil {
 		return err
 	}
-	currentVersion := currentClusterVersion.String()
-	latestVersion := kubernetes.LatestVersion().String()
-	fmt.Printf("Current Kubernetes cluster version: %s\n", currentVersion)
-	fmt.Printf("Latest Kubernetes version: %s\n", latestVersion)
-	fmt.Println()
-
 	nodeVersionInfoUpdate, err := upgradenode.UpdateStatus(nodeName)
 	if err != nil {
 		return err
 	}
+
+	fmt.Printf("Current Kubernetes cluster version: %s\n", currentClusterVersion.String())
+	fmt.Printf("Latest Kubernetes version: %s\n", kubernetes.LatestVersion().String())
+	fmt.Println()
+	fmt.Printf("Current Node version: %s\n", nodeVersionInfoUpdate.Current)
```

Comment:
```suggestion
	fmt.Printf("Current Node version: %s\n", nodeVersionInfoUpdate.Current.KubeletVersion.String())
```

Reply:
thanks updated, however for some reason the previous way showed exactly the same end result